xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/notification/rss/default.vm

```diff
@@ -0,0 +1,8 @@
+#template('notification/macros.vm')
+#set ($app  = "$!event.events.get(0).application")
+#if ($app == 'xwiki')
+    #set ($app = $services.localization.render('core.events.appName'))
+#end
+#set ($document = $xwiki.getDocument($event.document))
+$app:
+[$event.type] [$document.getRenderedTitle('plain/1.0')] $services.localization.render('notifications.events.by', ["#displayNotificationEventUsers($event.users, false)"])
```

Comment:
I wonder if you should not use the same template than the email plain text:
https://github.com/xwiki/xwiki-platform/blob/51f66d309694d07de0e09731aa81cec2381af335/xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/notification/email/default.plain.vm

It looks like this:
![plain text email](https://jira.xwiki.org/secure/attachment/34070/plainTextMail.png)